src/pdf/DayOverview/DayOverviewPdf.tsx

```diff
@@ -55,7 +56,13 @@ const styles = StyleSheet.create({
 
     cellLogo: { flex: 1 },
     cellName: { flex: 4 },
-    cellPostcode: { flex: 4 },
+    cellPostcode: {
+        flex: 4,
+        display: "flex",
+        flexDirection: "row",
+        alignItems: "flex-start",
+        paddingTop: "1px",
+    },
     cellTime: { flex: 2 },
```

Comment:
Made the postcode cell into a flex box in order to align both the text and the congestion charge logo at the top next to each other